<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerMappingDetailsClass</class>
 <widget class="QDialog" name="CustomerMappingDetailsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NA23 CustomerMappingDetailsVC</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>77</width>
     <height>26</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Recordset is saved with abandoning the dialog afterwards.</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSwitch">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="whatsThis">
    <string>Recordset is saved. Afterwards the dialog will be set back to record new ones.</string>
   </property>
   <property name="text">
    <string>Switch Adress</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>77</width>
     <height>26</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Closes the dialog without saving.</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_custExtern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>638</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>external customer data</string>
   </property>
   <widget class="QLineEdit" name="customernoextern">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>38</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_extern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>38</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_custnameExtern">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_custExtern">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customerno ext.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_custAdressExtern">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>62</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adress:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="street_extern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cip_extern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="location_extern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_custIntern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>638</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>internal customer data</string>
   </property>
   <widget class="QLineEdit" name="customernointern">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>38</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_intern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>38</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_custnameIntern">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_custIntern">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customerno int.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_custAdressExtern_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>62</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adress:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="street_intern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="location_intern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cip_intern">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>customernoextern</tabstop>
  <tabstop>customernointern</tabstop>
  <tabstop>btnSwitch</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>street_extern</tabstop>
  <tabstop>cip_extern</tabstop>
  <tabstop>location_extern</tabstop>
  <tabstop>name_extern</tabstop>
  <tabstop>name_intern</tabstop>
  <tabstop>street_intern</tabstop>
  <tabstop>location_intern</tabstop>
  <tabstop>cip_intern</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
